v22.1/row-level-ttl.md

```diff
@@ -0,0 +1,342 @@
+| `ttl_job_cron` <a name="param-ttl-job-cron"></a>         | Frequency at which the TTL job runs, specified using [CRON syntax](https://cron.help).                                                                                       | N/A                                 |
+| `ttl_automatic_column`                                   | If set, use the value of the `crdb_internal_expiration` hidden column.                                                                                                       | N/A                                 |
+
+For more information about TTL-related cluster settings, see [View TTL-related cluster settings](#view-ttl-related-cluster-settings).
+
+## Examples
+
+### Create a table with row-level TTL
+
+To specify a TTL when creating a table, use the SQL syntax shown below. For example, to create a new table with rows that expire after 15 minutes, execute a statement like the following:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE events (
+  id UUID PRIMARY KEY default gen_random_uuid(),
+  description TEXT,
+  inserted_at TIMESTAMP default current_timestamp()
+) WITH (ttl_expire_after = '3 minutes');
+~~~
+
+~~~
+CREATE TABLE
+~~~
+
+Insert some data; it should work as expected:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+INSERT INTO events (description) VALUES ('a thing'), ('another thing'), ('yet another thing');
+~~~
+
+~~~
+INSERT 3
+~~~
+
+### Add row-level TTL to an existing table
+
+To add a TTL to an existing table, use the SQL syntax shown below.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE events SET (ttl_expire_after = '90 days');
+~~~
+
+~~~
+ALTER TABLE
+~~~
+
+### View TTL jobs
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM [SHOW SCHEDULES] WHERE label LIKE 'row-level-ttl-%';
+~~~
+
+~~~
+          id         |       label       | schedule_status |        next_run        | state | recurrence | jobsrunning | owner |            created            |     command
+---------------------+-------------------+-----------------+------------------------+-------+------------+-------------+-------+-------------------------------+-------------------
+  745396332264194049 | row-level-ttl-114 | ACTIVE          | 2022-03-17 21:00:00+00 | NULL  | @hourly    |           0 | root  | 2022-03-17 20:01:48.033267+00 | {"tableId": 114}
+(1 row)
+~~~
+
+### Reset a storage parameter to its default value
+
+To reset a [TTL storage parameter](#ttl-storage-parameters) to its default value, use the [`ALTER TABLE`](alter-table.html) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE events RESET (ttl_job_cron);
+~~~
+
+~~~
+ALTER TABLE
+~~~
+
+### View TTL storage parameters on a table
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT relname, reloptions FROM pg_class WHERE relname = 'events';
```

Comment:
`SHOW CREATE TABLE` is also another valid way

Reply:
Updated to lead with an example with `SHOW CREATE TABLE`, left the other as an alternative